news-js/src/components/view/appView.ts

```diff
@@ -1,18 +1,30 @@
 import News from './news/news';
 import Sources from './sources/sources';
+import { IView }  from "./interfases/Iview";
+
+export interface IAppView{
+    news: IView,
+    sources: IView,
+    drawNews(data): void,
+    drawSources(data): void,
+};
+
+export class AppView implements IAppView{
+
+    news: IView;
+    sources: IView;
```

Comment:
```suggestion
    news: News;
    sources: Sources;
```
You don't have to create IView interface, since you can use class as type